src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php

```diff
@@ -164,22 +163,13 @@
             ->args([service(ContainerInterface::class)])
             ->tag('container.service_subscriber', ['id' => 'translator'])
             ->tag('kernel.cache_warmer')
-    ;
-
-    if (class_exists(LocaleSwitcher::class)) {
```

Comment:
We should be careful NOT doing conditional definition in the config files please. This logic is for DI extensions. Config files should remain purely declarative. /cc @symfony/mergers for future reviews.